server/src/main/java/io/spine/server/stand/SubscriptionCallback.java

```diff
@@ -47,8 +47,10 @@
      */
     static SubscriptionCallback forwardingTo(StreamObserver<SubscriptionUpdate> observer) {
         return update -> {
-            checkNotNull(update);
-            observer.onNext(update);
+            synchronized (observer) {
+                checkNotNull(update);
+                observer.onNext(update);
```

Comment:
Let's use a dedicated object to synchronize on.

Also, it's not like we promise timestamp-sorter delivery. We might as well use a non-fair reentrant lock. The promise of it is that the synchronization overhead is minimized.

From the codebase, it seems like the relationship between the `observer`s and `SubscriptionCallback`s is 1:1. So, it shouldn't matter if we choose to lock on the observer or an object (or a `Lock`) that belongs to the callback.